### DISCUSSION TITLE
Was BOM support removed?

### DISCUSSION THREAD

**Discussion by TheMrMilchmann** [Discord4J/Discord4J#863] [Q&A] [answered]
When looking at Central, it seems that BOM support was removed since the last BOM was published on 2020-11-02 (version `3.1.1`). However, after searching through the changelog I could not find any mention of the removal and now I wonder whether or not it is intentional or a bug.

Compare https://repo1.maven.org/maven2/com/discord4j/bom/ and https://repo1.maven.org/maven2/com/discord4j/discord4j-core/.

**Comment by quanticc** [comment, marked as answer]
sorry about that, It's still supported but we haven't pushed the latest tags yet. I'll work on that now

Update: BOM 3.1.3 should now be available on Maven Central

https://repo1.maven.org/maven2/com/discord4j/bom/3.1.3/

There's also a 3.1.4-SNAPSHOT bom for the latest builds from the 3.1.x branch and dependencies

https://oss.sonatype.org/content/repositories/snapshots/com/discord4j/bom/3.1.4-SNAPSHOT/
